### DISCUSSION TITLE
Disable ankle rotation IK?

### DISCUSSION THREAD

**Discussion by xkamronx** [Sixze/ALS-Refactored#336] [Q&A]
Can you disable the ankle rotation IK without removing the foot adjusting height?

There's a lot of situations where it looks very rubbery and the characters ankles look horrifically broken.

**Comment by Sixze** [comment]
I've added the customizable foot rotation limits in 89719a0ea2782eb0453baadc324a710db5f0932a, now the foot will assume a more natural pose when on highly sloped surfaces.
